autometrics-ts.code-workspace

```diff
@@ -0,0 +1,12 @@
+{
+	"folders": [
+		{
+			"path": "."
+		}
+	],
+	"settings": {
+		"editor.defaultFormatter": "rome.rome",
+		"editor.formatOnSave": true,
+		"typescript.tsdk": "node_modules/typescript/lib"
+	}
+}
```

Comment:
I think this shouldn't be part of the repository.

Reply:
ah yeah, i'll remove